dsls/sql/src/main/java/org/apache/beam/dsls/sql/transform/BeamAggregationTransform.java

```diff
@@ -19,13 +19,23 @@
 
 import java.io.Serializable;
```

Comment:
Change the name of class to something like `BeamAggregationTransforms` might be better? Since it is not a `transform`, it is a `collection` of tranforms and dofns.